From 39fcde3d33049733744ee8abe4ef730b3f4e3f89 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Wed, 3 Nov 2021 11:03:25 -0600
Subject: [PATCH 1/5] Bump Security String to 2021-11-05

Implemented:
============
CVE:            References:  Type:  Severity:  Updated AOSP versions:
CVE-2021-0919   A-197336441  DoS    Moderate   9, 10, 11
CVE-2021-0921   A-195962697  EoP    High       11
CVE-2021-0922   A-195630721  EoP    Moderate   11
CVE-2021-0926   A-191053931  EoP    High       9, 10, 11, 12
CVE-2021-0927   A-189824175  EoP    High
CVE-2021-0928   A-188675581  EoP    High       9, 10, 11
CVE-2021-0930   A-181660091  RCE    Critical   9, 10, 11, 12
CVE-2021-0931   A-180747689  ID     High       9, 10, 11, 12
CVE-2021-0933   A-172251622  EoP    High       9, 10, 11, 12

Previously Implemented:
=======================
CVE:            References:  Type:  Severity:  Updated AOSP versions:  Prior Change:
CVE-2020-13871  A-192606047  ID     High       11                      1935111
CVE-2021-0434   A-167403112  EoP    High       9, 10, 11               18c83959b1
CVE-2021-0649   A-191382886  EoP    High       11                      0974700666a2
CVE-2021-0650   A-190286685  ID     High       9, 10, 11               be8ae5a
CVE-2021-0653   A-177931370  ID     High       9, 10, 11               bfad9c3e8e1e

Not Implemented:
================
None

Not Applicable (platform source):
=================================
CVE:            References:  Type:  Severity:  Updated AOSP versions:
CVE-2021-0799   A-197647956  EoP    High       12
CVE-2021-0918   A-197536150  RCE    Critical   12
CVE-2021-0923   A-195338390  EoP    High       12
CVE-2021-0925   A-191444150  ID     High       12
CVE-2021-0932   A-173025705  EoP    High       10

Change-Id: I592893069ab0a25f30b734bb6c922a8af7dd86d2
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 4a00f443c..c19f61970 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -240,7 +240,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-      PLATFORM_SECURITY_PATCH := 2021-10-01
+      PLATFORM_SECURITY_PATCH := 2021-11-05
 endif
 .KATI_READONLY := PLATFORM_SECURITY_PATCH
 
-- 
2.25.1

